apps/datahub/src/app/record/record-metadata/record-metadata.component.html

```diff
@@ -120,9 +120,13 @@
     </div>
 
     <div *ngIf="displayApi$ | async">
-      <datahub-record-apis class="block mt-6"></datahub-record-apis>
+      <datahub-record-apis
+        (displayCustomPanel)="isCustom($event)"
```

Comment:
Note that in the long run, there will be several kinds of APIs forms. For instance
- OGC API Features
- WFS
- IGN download service
- IGN extraction service

Reply:
Noted. This output is here to allow the custom API cards to open the panel to customize the URL, which isn't available for pre-defined API cards.